qa/standalone/ceph-helpers.sh

```diff
@@ -34,10 +35,12 @@ fi
 
 if [ `uname` = FreeBSD ]; then
     SED=gsed
+    AWK=gawk
```

Comment:
could you point me to the place(s) we are using a feature where only gawk supports?

Reply:
@tchaikov 
lines 1438 or 1448, where there are comparisons in the evaluation.
`awk` gives an error, `gawk` does the expected thing.

Reply:
are they 

https://github.com/ceph/ceph/blob/821b6b486cebbaa26ad4147807f0b08775055b99/qa/standalone/ceph-helpers.sh#L1438

and 

https://github.com/ceph/ceph/blob/821b6b486cebbaa26ad4147807f0b08775055b99/qa/standalone/ceph-helpers.sh#L1448

?

Reply:
@tchaikov 
No the calls to call, Got to check the gitrepo, since mine are obviously skewed:
1433: `while test "$(calc $total + $i \<= $timeout)" = "1"; do`
and
1445: `if test "$(calc $total \< $timeout)" = "1"; then`

Reply:
ic, so it's https://github.com/ceph/ceph/blob/821b6b486cebbaa26ad4147807f0b08775055b99/qa/standalone/ceph-helpers.sh#L1411

turns out we trade `bc` for `gawk` in https://github.com/ceph/ceph/pull/21101/commits/ce9c029858d8deb249afe8effd5d80ae623f69a2

Reply:
@tchaikov 
Yup, I think `bc` is a beter calculator, but then like you say: 'so it is'
And I don't feel redoing this for a third time.